examples/task.hpp

```diff
@@ -268,6 +268,12 @@ class basic_task {
     __coro::coroutine_handle<_promise> coro_;
     std::optional<awaiter_context_t<_promise, ParentPromise>> context_{};
 
+    ~_task_awaitable() {
+        if (_coro) {
+            _coro.destroy();
+        }
+    }
```

Comment:
member name and formatting.

```suggestion
    ~_task_awaitable() {
      if (coro_)
        coro_.destroy();
    }
```